### PR TITLE
update decisions.md

### PR DESCRIPTION
**Pull request by rpottsoh**
Corrected some MD.  Also modified last sentence.

### CODE REVIEW

File: language-tracks/launch/decisions.md [hunk 1]
```diff
@@ -57,13 +57,13 @@ Remember that all Exercism exercises are by nature very small, so there's no nee
...
+1. If the command to run the tests is different for different exercises, we want to be able to generate the command.
+1. The Exercism tooling needs to be able to identify the test file and reference solution.
 
 In Exercism we want to be as true to the language as possible, since we're aiming to get people conversant with the common idioms and practices as possible.
 
 In some programming languages it might not make sense to talk about "test file" and "solution file".
 That's probably fine, but it might require us to jump through some hoops to get everything working.
 
-In terms of naming, though, as long as it's consistent, we should be fine.
+The naming convention must be consistent.
```
Comment (kytrinyx) [inline language-tracks/launch/decisions.md]: If we're going to state this as emphatically as this, then maybe it should go into the bulleted list above. What do you think?

Comment (rpottsoh) [reply language-tracks/launch/decisions.md]: I like the bullet idea.  I can replace `must` with `should`.

**Comment by rpottsoh** [comment]
@kytrinyx I'll go ahead and squash and merge if you are fine with this update.

**Comment by kytrinyx** [comment]
Yepp that sounds good

**Pull request merged**